src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -1048,6 +1053,11 @@ namespace winrt::TerminalApp::implementation
             _mruPanes.insert(_mruPanes.begin(), paneId.value());
         }
 
+        if (_rootPane->GetLeafPaneCount() == 1)
+        {
+            _closePaneMenuItem.Visibility(WUX::Visibility::Collapsed);
+        }
+

```

Comment:
I know the feature request was to display this when multiple panes were open. Is there any reason for this? Personally, I'd prefer to just have the item always visible.

If we decide that that's the change we want to go with, you can remove this block of code and remove any maintenance/reference to `_closePaneMenuItem`.